docs/design/2020-09-30-index-usage-information.md

```diff
@@ -0,0 +1,150 @@
+# Proposal:
+
+- Author(s):     [rebelice](https://github.com/rebelice)
+- Last updated:  Sep. 30, 2020
+- Discussion at: 
+
+## Abstract
+
+This document describes the design of recording index usage information.
+
+## Background
+
+There may be unused indexes in the database. In addition, modifying database tables, indexes, and query statements may cause some indexes to not be used in the future. Users may want to clear these long-unused indexes to relieve storage and performance pressure.
+source: [issue/14998](https://github.com/pingcap/tidb/issues/14998) [issue/14607](https://github.com/pingcap/tidb/issues/14607) [issue/17508](https://github.com/pingcap/tidb/issues/17508) [asktug](https://asktug.com/t/topic/34614/12)
+
+## Proposal
+
+### SCHEMA_INDEX_USAGE
+
+Design system tables to record index usage information. The system table is designed as follows:
+1. The system table is located in mysql database and is named SCHEMA_INDEX_USAGE
+2. Columns of SCHEMA_INDEX_USAGE
+| Column name  | Data type | Description |
+|  ---------   | --------  | ----------- |
+| TABLE_SCHEMA | varchar   | Name of the database on which the table or view is defined.|
+| TABLE_NAME   | varchar   | Name of the table or view on which the index is defined.|
+| INDEX_NAME   | varchar   | Name of the index.|
+| QUERY_COUNT  | longlong  | Number of the SQL using this index.|
+| ROWS_SELECTED| longlong  | Number of rows read from the index. We can check the selectivity of the index through `ROWS_READ` / `QUERY_COUNT`.|
+| LAST_USED_AT | timestamp | The last time of the SQL using this index.|
+3. User interface：`select * from mysql.SCHEMA_INDEX_USAGE`
+
+#### Table creation: 
+
+```sql
+create table SCHEMA_INDEX_USAGE (
+	TABLE_SCHEMA varchar(64),
+	TABLE_NAME varchar(64),
+	INDEX_NAME varchar(64),
+	QUERY_COUNT BIGINT,
+	ROUWS_SELECTED BIGINT,
+	LAST_USED_AT timestamp,
+	PRIMARY KEY (SCHEMA_NAME, TABLE_NAME, INDEX_NAME)
+);
+```
+
+Because the max length of schema name, table name and index name is 64, so we use `varchar(64)`. The documentation is [here](https://dev.mysql.com/doc/refman/5.7/en/identifier-length.html).
```

Comment:
could you change the link to TiDB user doc?

Reply:
I did not find relevant information in TiDB user doc.